milestone-5/backend/index.js

```diff
@@ -0,0 +1,96 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+app.use(express.json());
+const mongoose = require("mongoose");
+const Recipe = require("./models/recipeSchema");
+const connection_url =
+  "mongodb+srv://new-user:h4i_bootcamp@cluster0.yhfnc.mongodb.net/RecipesDB?retryWrites=true&w=majority";
+
+mongoose
+  .connect(connection_url)
+  .then(() => console.log("Successfully connected"))
+  .catch((error) => console.error(`Could not connect due to ${error}`));
+
+app.use(cors());
+
+app.listen(3000, function () {
+  console.log("CORS-enabled web server listening on port 3000");
+});
```

Comment:
Is there a specific reason you tried to use this CORS stuff. On my end it would not work with this active, but deleting this app.listen made it run fine. If this was just for debugging/testing you may want to use a different port